lib/exabgp/bgp/message/update/nlri/cidr.py

```diff
@@ -72,7 +72,7 @@ def prefix (self):
 		return "%s/%s" % (self.top(),self.mask)
 
 	def index (self):
-		return character(self.mask) + str(self._packed[:CIDR.size(self.mask)])
+		return chr(self.mask) + str(self._packed[:CIDR.size(self.mask)])
```

Comment:
I think that index returns a string rather than a bytestring, and that we need a chr rather than a byte to allow concatenation with str(self._packed ...).